apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/http/HttpParameterParserUtil.java

```diff
@@ -433,7 +433,7 @@ public static Shuffling getShuffling(HttpServletRequest req) throws ParameterExc
     }
 
     public static long getQuantityATU(HttpServletRequest req) throws ParameterException {
-        return getLong(req, "quantityATU", 1L, Constants.MAX_ASSET_QUANTITY_ATU, true);
+        return getLong(req, "quantityATU", 1L, lookupBlockchainConfig().getInitialSupply() * lookupBlockchainConfig().getOneAPL(), true);
```

Comment:
Use Math.multiplyExact to avoid overflow.

Reply:
fixed